inst/htmlwidgets/r3dmol.yaml

```
@@ -4,10 +4,10 @@ dependencies:
     src: htmlwidgets/lib/jquery-3.5.1
     script: jquery.min.js
   - name: r3dmol
-    version: 1.5.4
-    src: htmlwidgets/lib/3Dmol-1.5.4
+    version: 1.6.2
+    src: htmlwidgets/lib/3Dmol-1.6.2
```

```suggestion
    src: htmlwidgets/lib/3Dmol
```
I think it would be better to remove the version in directory name, in order to avoid to rename it in every upgrade.